src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/GetCaseResponse.java

```diff
@@ -38,8 +35,8 @@
     @JsonProperty("reference")
     private String reference;
 
-    @JsonRawValue
-    private String data;
+    @JsonProperty("data")
```

Comment:
I think this is fine, but it needs smoke testing before while this is a draft PR